examples/seismic/tutorials/16_ader_fd.ipynb

```diff
@@ -5,7 +5,7 @@
    "id": "31226b14-e907-4503-9918-a09163363217",
```

Comment:
<p>Considering you derive the equation for constant <code>c</code> not sure should do that </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/2450/discussion/#comment-1758172535'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>Assuming locally-constant material parameters is an approximation that you can make to simplify the equations and avoid dealing with chain rule. It's still going to work, but yeah, you can incur errors here</p>